### PR TITLE
[artifactory] documentation updates

### PR DESCRIPTION
**Pull request by aaleksan20**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] CHANGELOG.md updated
- [X] Variables and other changes are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by aaleksan20** [comment]
Signing the CLA

**Comment by chukka** [comment]
@aaleksan20 Seems like Readme file is missing

File: stable/artifactory/README.md [hunk 1]
```diff
@@ -1,1160 +1,101 @@
 # JFrog Artifactory Helm Chart
 
-**Heads up: Our Helm Chart docs are moving to our main documentation site. For Artifactory installers, see [Installing Artifactory](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory).**
+**Our Helm Chart docs have moved to our main documentation site. For JFrog Artifactory installers, see [Installing Artifactory](https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory).**
```
Comment (chukka) [inline stable/artifactory/README.md:3]: @aaleksan20 should we directly point to helm documentation in wiki ? - https://www.jfrog.com/confluence/display/JFROG/Installing+Artifactory#InstallingArtifactory-HelmInstallation

Comment (aaleksan20) [reply stable/artifactory/README.md:3]: Yes, good idea

**Comment by aaleksan20** [comment]
Closing until can update

**Pull request closed without merging**